tests/test_indexes.py

```diff
@@ -154,29 +152,28 @@ def test_index_troughtables_creation():
     data = test_data
     parent_schema = SchemaType(data)
     dataset_schema = DatasetSchema(parent_schema)
-    DATABASE_URL = env("DATABASE_URL", None)
-    db_url = DATABASE_URL
-    engine = _get_engine(db_url)
     indexes_name = []
 
     for table in data["tables"]:
 
         importer = BaseImporter(dataset_schema, engine)
         # the generate_table and create index
-        importer.generate_db_objects(table["id"], ind_tables=True, ind_extra_index=True)
+        importer.generate_db_objects(
+            f'{table["id"]}',
+            ind_tables=True,
+            ind_extra_index=True,
+        )
 
     for table in data["tables"]:
 
         dataset_table = dataset_schema.get_table_by_id(table["id"])
 
         for table in dataset_table.get_through_tables_by_id():
 
-            CONN = create_engine(DATABASE_URL, client_encoding="UTF-8")
+            CONN = create_engine(engine.url, client_encoding="UTF-8")
```

Comment:
Namen van variabele in all caps betekent in python dat het constanten zijn, maar dat is hier niet het geval, ik zou er gewoon lowercase (snakecase) van maken.